src/components/MessageList/VirtualizedMessageList.tsx

```diff
@@ -327,6 +361,8 @@ export function VirtualizedMessageList<
 
   return (
     <VirtualizedMessageListWithContext
+      //@ts-expect-error
```

Comment:
assuming this is because channel is optionally undefined?

Reply:
Question: Is rendering a list without a channel a valid scenario? If not, I would rather terminate this possible timeline as early as possible rather than optioning variables further down.

Reply:
It should never happen, but the channel is listed as optional because there are two different ways it can be set, either through props (not using ChannelList) or through context (using ChannelList). So both of these channel types are optional since it can happen either way, but one of them should exist. Looks like we just need to adjust the ChannelInner props to not expect a partial `channel` type, since we early return before this render anyways if the channel doesn't exist